File: AppLovin-Android-Demo-App/src/main/java/com/applovin/apps/demoapp/nativeads/NativeAdDemoMenuActivity.java
package com.applovin.apps.demoapp.nativeads;

import android.content.Intent;
import android.graphics.Color;
import android.text.Html;
import android.view.Gravity;
import android.widget.TextView;

import com.applovin.apps.demoapp.DemoMenuActivity;
import com.applovin.apps.demoapp.DemoMenuItem;

public class NativeAdDemoMenuActivity
        extends DemoMenuActivity
{

    @Override
    protected void setupListViewFooter()
    {
        TextView footer = new TextView( getApplicationContext() );
        footer.setPadding( 0, 20, 0, 0 );
        footer.setGravity( Gravity.CENTER );
        footer.setTextSize( 18 );
        footer.setTextColor( Color.RED );
        footer.setText( Html.fromHtml( "You must turn <b>ON</b> \" Native Ads\" in your AppLovin dashboard" ) );

        listView.addFooterView( footer );
        listView.setFooterDividersEnabled( false );
    }

    @Override
    protected DemoMenuItem[] getListViewContents()
    {
        DemoMenuItem[] result = {
                new DemoMenuItem( "Single ad", "Programmatically load an ad using our open-source carousel view", new Intent( this, NativeAdCarouselUIActivity.class ) ),
                new DemoMenuItem( "Multiple ads", "Simple native ads in a RecyclerView", new Intent( this, NativeAdRecyclerViewActivity.class ) )
        };
        return result;
    }
}

File: AppLovin-Kotlin-Demo-App/src/main/java/com/applovin/apps/kotlindemoapp/nativeads/NativeAdDemoMenuActivity.kt
package com.applovin.apps.kotlindemoapp.nativeads

import android.content.Intent
import com.applovin.apps.kotlindemoapp.DemoMenuActivity
import com.applovin.apps.kotlindemoapp.DemoMenuItem

class NativeAdDemoMenuActivity : DemoMenuActivity()
{
    override fun getListViewContents(): Array<DemoMenuItem> = arrayOf(
            DemoMenuItem("Single ad", "Programmatically loading an ad using our open-source carousel view", Intent(this, NativeAdCarouselUIActivity::class.java)),
            DemoMenuItem("Multiple ads", "Simple native ads in a RecyclerView", Intent(this, NativeAdRecyclerViewActivity::class.java))
    )
}
